src/components/UI/ToggleSwitch/index.tsx

```diff
@@ -28,7 +34,7 @@ export default function ToggleSwitch(props: Props) {
         aria-label={title}
       />
 
-      <span {...props} className="checkmark" />
+      <span {...checkmarkProps} className="checkmark" />
```

Comment:
What is this change doing exactly?
I believe that if we pass a prop that is not listed, it will be ignored.
so we need to check the props we use when calling this component :)

Reply:
```javascript
const {
    handleChange,
    value,
    disabled,
    title,
    dataTestId = 'toggleSwitch',
    ...rest
  } = props
```
are by any chance thinking of something like `rest` property? Because destructuring an object doesn't remove destrustured properties.
anyway, I removed properties which caused errors: `handleChange` and `disabled`, but I kept the rest just in case the attributes are used somewhere and I didn't want to break anything.